cmd/neofs-cli/modules/container/create.go

```diff
@@ -50,9 +50,15 @@ It will be stored in sidechain when inner ring will accepts it.`,
 			common.ExitOnErr(cmd, "unable to get netmap snapshot to validate container placement, "+
 				"use --force option to skip this check: %w", err)
 
-			_, err = resmap.NetMap().ContainerNodes(*placementPolicy, nil)
+			nodesByRep, err := resmap.NetMap().ContainerNodes(*placementPolicy, nil)
 			common.ExitOnErr(cmd, "could not build container nodes based on given placement policy, "+
 				"use --force option to skip this check: %w", err)
+
+			for i, nodes := range nodesByRep {
+				if placementPolicy.ReplicaNumberByIndex(i) > uint32(len(nodes)) {
+					common.ExitOnErr(cmd, "", fmt.Errorf("not enough nodes in selector for replicas"))
```

Comment:
Can we also give some numbers in the error message?

Reply:
How bout this: `the number of nodes '%d' is not enough for the number of replicas '%d'`